tests/integration/test_enrollment/data/wazuh_conf.yaml

```diff
@@ -0,0 +1,27 @@
+---
```

Comment:
We should rename the file with a more descriptive name: `wazuh_test_name_conf.yaml`

Reply:
Fixed